polaris.shopify.com/src/components/IconDetails/IconDetails.tsx

```diff
@@ -15,131 +14,128 @@ interface Props {
 }
 
 function IconDetails({ fileName, iconData }: Props) {
-  if (!fileName) {
-    return (
-      <div className={styles.SidebarCard}>
-        <div className={styles.SidebarEmptyState}>
-          <span style={{ opacity: 0.25 }}>
-            <Image
-              width={20}
-              height={20}
-              src="/icons/BehaviorMajor.svg"
-              alt="Cursor suggesting to click an item"
-            />
-          </span>
-          <p>Select an icon</p>
-        </div>
-      </div>
-    );
-  }
+  if (!fileName) return <EmptyState />;
 
   const { set, description, name, keywords } = iconData;
+
+  const reactExamples = {
+    imports: `import {\n  ${fileName}\n} from '@shopify/polaris-icons';`,
+    componentUsage: `<Icon\n  source={${fileName}}\n  color="base"\n/>`,
```

Comment:
I noticed that I accidentally messed up the indentation of these examples when I was working in this file. It's easy to indent a section and not realize that the examples get indented too.

Inlining this code looks a bit ugly, but helps prevent accidental indentation changes.